certified-connectors/Sociabble/apiDefinition.swagger.json

```diff
@@ -724,6 +722,54 @@
         "summary": "Upload a media",
         "description": "Upload a media to use during the creation of an internal news",
         "operationId": "UploadMedia",
+        "deprecated": true,
+        "x-ms-api-annotation": {
+            "status": "Production",
```

Comment:
@mdujardibs Can you please let us know why you want to publish your operations with `Production`? It is not allowed to publish new connector or its operations with `Production` tag. Please replace it with `Preview`.

All the new connectors are deployed with Preview tag. In order to remove the tag, we require your connector to meet the following criteria:

0. Your connector needs to be certified and published in all the production regions.
1. Connector has 99.95% reliability (calculated by % of 5xx (-502 & -504) errors) 
2. Connector has 99.5% SLO (percentage of connections meeting SLA bar above)
3. Connector has greater than 80% success rate (Calculated by % of 4xx and 5xxx errors) 
4. Connector has at least 500 weekly active users over the last 3 weeks.
5. Backend service APIs, that the connector is calling, needs to be in PRODUCTION.
6. Partner should be able provide 24/7 support for high severity incidents.

If you feel your connector meet above criteria, please start an email thread with [us](mailto:ConnectorPartnerMgmtTeam@service.microsoft.com) to remove the "Preview" tag.

Reply:
@mdujardibs Applies the same comment to all the similar properties in the swagger.